packages/entities/entities-routes/package.json

```diff
@@ -71,11 +71,8 @@
   },
   "dependencies": {
     "@kong-ui-public/entities-shared": "workspace:^",
+    "@kong-ui-public/expressions": "workspace:^",
```

Comment:
This is incorrect; you’re externalizing the dependencies in the Vite config, but then listing them as direct dependencies in the package.json?